mdoc-js/src/main/scala/mdoc/modifiers/JsModifier.scala

```diff
@@ -165,7 +165,9 @@ class JsModifier extends mdoc.PreModifier {
     } else {
       val output = MemOutputFile.apply()
 
-      val linking = linker.link(virtualIrFiles ++ sjsir, Nil, LinkerOutput.apply(output), sjsLogger)
+      val currentLinker = if (config.reuseLinker) linker else newLinker()
```

Comment:
Does the linker have some shutdown/cleanup method that we should call when we initialize a new linker?

Reply:
I'm not seeing any in the interface, but there's something like `ClearableLinker` that seems to just manage an internal `_linker` state and setting it to `null` when `.clear()` is called

I'll check with Scala.js folks on whether I should use that, or just set batch mode on the linker which should achieve the same results without this if condition

>  just sent you a collaborator invitation

Thank you, appreciate it :)

Reply:
321db8a should be better - clearable linker + batch mode means no manual management of linker lifecycle and the same results